### PR TITLE
Pick task location

### PR DESCRIPTION
**Pull request by Polymeth**
The map can display markers however there was no way to select a marker while creating an event, so this PR adds a map picker in Create Event menu.

Will need to add one on Edit Event later on.

This PR addresses #201.

### CODE REVIEW

**Review by sarahbadr17 — COMMENTED** [review]
Just some few comments to remove otherwise very good! It works well on my phone! Good job I know it wasn't easy :)

File: app/src/main/java/com/github/se/assocify/ui/screens/event/tasktab/task/TaskViewModel.kt [hunk 1]
```diff
@@ -268,6 +272,7 @@ data class TaskState(
     val time: String = "",
     val duration: String = "",
     val event: Event? = null,
+    val location: String = "",
```
[the remark this answers is not in the document]
Comment (Polymeth) [reply app/src/main/java/com/github/se/assocify/ui/screens/event/tasktab/task/TaskViewModel.kt:275]: Good idea :)

[the remark this answers is not in the document]
Comment (Polymeth) [reply app/src/main/java/com/github/se/assocify/ui/screens/event/tasktab/task/TaskViewModel.kt:275]: nvm I think we said we didn't want all task to have a location

**Review by sarahbadr17 — APPROVED** [review]
Incredible job! Worthy of a senior programmer

**Pull request merged**